server/game.py

```diff
@@ -310,32 +310,29 @@ def add_train(self, nickname):
                 train_color,
                 self.handle_train_death,
                 self.config.tick_rate,
+                self.config.reference_tick_rate
             )
             self.update_passengers_count()
             return True
         return False
 
-    def send_cooldown(self, nickname, death_reason):
+    def send_respawn_cooldown(self, nickname, death_reason):
         """Remove a train and update game size"""
```

Comment:
This comment might be incorrect. The game size doesn't change when a train dies.